core/loadpoint_vehicle.go

```diff
@@ -100,80 +100,77 @@ func (lp *Loadpoint) selectVehicleByID(id string) api.Vehicle {
 
 // setActiveVehicle assigns currently active vehicle, configures soc estimator
 // and adds an odometer task
-func (lp *Loadpoint) setActiveVehicle(vehicle api.Vehicle) {
+func (lp *Loadpoint) setActiveVehicle(v api.Vehicle) {
 	lp.vehicleMux.Lock()
-	if lp.vehicle == vehicle {
-		lp.vehicleMux.Unlock()
-		return
-	}
 
 	from := "unknown"
 	if lp.vehicle != nil {
 		lp.coordinator.Release(lp.vehicle)
 		from = lp.vehicle.Title()
 	}
 	to := "unknown"
-	if vehicle != nil {
-		lp.coordinator.Acquire(vehicle)
-		to = vehicle.Title()
+	if v != nil {
+		lp.coordinator.Acquire(v)
+		to = v.Title()
 	}
 
-	lp.vehicle = vehicle
+	lp.vehicle = v
 	lp.vehicleMux.Unlock()
 
-	lp.log.INFO.Printf("vehicle updated: %s -> %s", from, to)
+	if from != to {
+		lp.log.INFO.Printf("vehicle updated: %s -> %s", from, to)
+	}
 
-	// lock api
+	// reset session values
 	lp.Lock()
-
-	// reset minSoc and targetSoc before change
-	lp.setMinSoc(0)
-	lp.setTargetSoc(100)
-
-	// reset target energy
-	lp.setTargetEnergy(0)
-
-	// unlock api
+	lp.setLimitSoc(0)
```

Comment:
Das führt beim Neustart dazu, dass der Limit Soc Wert aus den Settings mit 0 überschrieben wird und damit quasi nutzlos ist.

Ich sehe hier zwei Möglichkeiten:

1. Reset (= 0) nur beim aktiven Wechsel und nicht beim Hochfahren ausführen
2. Persistieren der Limits (energy & soc) komplett entfernen, weil nicht funktional.

Ich würde natürlich 1. präferieren, weiß aber nicht, wie wir das elegant umsetzen können. Die zweite Lösung ist einfacher aber natürlich nervig, wenn man bspw. nen Stromausfall hatte und das Limit dadurch aufgehoben wird. Durch unsere Trennung von limit__ und plan__ und dem limitSoc am Fahrzeug ist, das allerdings nicht mehr so kritisch wie es bislang war. Die Plan-Werte und die Werte am Fahrzeug bleiben ja weiterhin persistent.

@andig Vorschlag: Wenn du einen einfachen Weg für 1. findest, lass uns den umsetzen. Falls wir das nur sehr umständändlich hinbekommen wär ich dafür, die Persistenz der beiden Werte (limitEnergy, limitSoc) am LP aufzugeben.

Reply:
Ich hab jetzt erstmal 2. umgesetzt. Keine Persistenz für Session Limits.

Reply:
@naltatis hier ist noch etwas krumm bzw. unnötig. Du beziehst Dich ja auf `setActiveVehicle`. Aktuell passiert das zusätzlich _auch_ noch in `evVehicleDisconnectHandler`. Letzteres könnte ok sein, abhängig von unsere Erwartung: sollte zurück gesetzt werden, wenn wir das Fahrzeug im UI auswählen, ohne dass es einen (erkannten) Disconnect gab? Falls es den Disconnect gab wären die Werte zu diesem Zeitpunkt eh schon weg.
Wenn Du der Argumentation folgst müsste das "Auf null setzten" nur im `setActiveVehicle` raus. 
Im `evVehicleDisconnectHandler` könnten wir das noch vereinfachen und die privaten Methoden verwenden, am Ende dann `lp.requestUpdate()`. Das sollten wir beim Connect dann auch gleich tun.

Reply:
> sollte zurück gesetzt werden, wenn wir das Fahrzeug im UI auswählen, ohne dass es einen (erkannten) Disconnect gab?

Ja, beim Fahrzeug ändern muss mMn nicht zurückgesetzt werden. Allerdings müssen wir vmtl. "aufräumen": limitSoc entfernen, wenn man von einem Online- auf ein Offline-Fahrzeug wechselt und limitEnergy entfernen, wenn Offline > Online. Vermute, dass sonst die Regelung Quatsch macht? Aber ich weiß es auch nicht genau. Die UI kommt da mMn mit klar.

Reply:
Falls das kein Problem ist, bin ich da voll bei dir und wir sollten das zurücksetzen erst beim disconnect machen.

Reply:
Sollte gehen (tm). Wir wählen ja auh in Go immer das fürs Fahrzeug relevante Limit aus.

Reply:
Dann wäre jetzt schon alles wie gewünscht?